Monotype/c2pa-rs
# C2PA-55: Use of the `C2PA` font table

# JIRA Issue

- https://monotype.atlassian.net/browse/C2PA-55

# Checklist


# Notes for Reviewers




Updates to the latest version `0.19.1` from upstream and adds support for `C2PA` font table for storing the manifest store and uses the remote manifest uri. The changes support file and stream support. I'm still not sure how to get the remote manifest URI, but it is embedded as well.

Note, this also adds support for remote manifests URIs.

# Verification Instructions


Can you still sign a font?

Can you add the URI to the font?

Use the changes in https://github.com/Monotype/c2patool/pull/4 for verifying.

Since the `"otf"` feature is optional and off by default, to run the unit tests added you will need to run one of the following:

- `cargo test --lib asset_handlers::otf_io --features "otf, file_io"` - to run only the `otf_io` unit tests
- `cargo test --all-features` - to run all unit tests, including `otf_io` unit tests.

> Actively maintained by the @Monotype/driverpdldev team.
